### ISSUE TITLE
User shield doesn't get green after verification

### ISSUE THREAD

**Issue by jurgenhaas** [vector-im/element-web#15553]
### Description

Started a new conversation with another user in Element and started the verification process. We both selected to verify by comparing emojis and we both confirmed that they were similar. We hen go a confirmation message, that the other person is verified.

Problem being, the gray shield remains gray.

Expectation: that shield should turn green.

**Comment by jryans** [comment]
@jurgenhaas Is there still a verify button for the user as well, or has that disappeared? Please submit debug logs (top left menu -> Settings -> Help & About -> Submit debug logs) and reference this issue so we can investigate further.

**Comment by jurgenhaas** [comment]
@jryans Thanks for getting back on this one. Yes, the link to verify that user was still there although the chat showed a message that this user was verified.

That user and myself both had an account at matrix.org and the other user reported, that he had that issue with all users.

Since I reported this issue, the other user created a new account at tchncs.de and we tried there again, where it worked as expected: the shield turned green and the verify link disappeared.

I'll get back o that user to see if he's still up for further tests with the original account so that we could provide the debug logs.
